exporters/otlp/logs/src/test/java/io/opentelemetry/exporter/otlp/logs/OtlpGrpcLogsExporterTest.java

```diff
@@ -154,257 +147,8 @@ void invalidConfig() {
-    try {
-      assertThat(exporter.export(Collections.singletonList(generateFakeLog())).isSuccess())
-          .isFalse();
-    } finally {
-      exporter.shutdown();
-    }
-  }
-
-  @Test
-  void testExport_ResourceExhausted() {
-    fakeCollector.setReturnedStatus(Status.RESOURCE_EXHAUSTED);
-    OtlpGrpcLogExporter exporter =
-        OtlpGrpcLogExporter.builder().setChannel(inProcessChannel).build();
-    try {
-      assertThat(exporter.export(Collections.singletonList(generateFakeLog())).isSuccess())
-          .isFalse();
-    } finally {
-      exporter.shutdown();
-    }
-  }
-
-  @Test
-  void testExport_OutOfRange() {
-    fakeCollector.setReturnedStatus(Status.OUT_OF_RANGE);
-    OtlpGrpcLogExporter exporter =
-        OtlpGrpcLogExporter.builder().setChannel(inProcessChannel).build();
-    try {
-      assertThat(exporter.export(Collections.singletonList(generateFakeLog())).isSuccess())
-          .isFalse();
-    } finally {
-      exporter.shutdown();
-    }
-  }
-
-  @Test
-  void testExport_Unavailable() {
-    fakeCollector.setReturnedStatus(Status.UNAVAILABLE);
-    OtlpGrpcLogExporter exporter =
-        OtlpGrpcLogExporter.builder().setChannel(inProcessChannel).build();
-    try {
-      assertThat(exporter.export(Collections.singletonList(generateFakeLog())).isSuccess())
-          .isFalse();
-    } finally {
-      exporter.shutdown();
-    }
-    LoggingEvent log =
-        logs.assertContains(
-            "Failed to export logs. Server is UNAVAILABLE. "
-                + "Make sure your collector is running and reachable from this network.");
-    assertThat(log.getLevel()).isEqualTo(Level.ERROR);
-  }
-
-  @Test
-  void testExport_Unimplemented() {
-    fakeCollector.setReturnedStatus(Status.UNIMPLEMENTED);
-    OtlpGrpcLogExporter exporter =
-        OtlpGrpcLogExporter.builder().setChannel(inProcessChannel).build();
-    try {
-      assertThat(exporter.export(Collections.singletonList(generateFakeLog())).isSuccess())
-          .isFalse();
-    } finally {
-      exporter.shutdown();
-    }
-    LoggingEvent log =
-        logs.assertContains(
-            "Failed to export logs. Server responded with UNIMPLEMENTED. "
-                + "This usually means that your collector is not configured with an otlp "
-                + "receiver in the \"pipelines\" section of the configuration. "
-                + "Full error message: UNIMPLEMENTED");
-    assertThat(log.getLevel()).isEqualTo(Level.ERROR);
-  }
-
-  @Test
-  void testExport_DataLoss() {
-    fakeCollector.setReturnedStatus(Status.DATA_LOSS);
-    OtlpGrpcLogExporter exporter =
-        OtlpGrpcLogExporter.builder().setChannel(inProcessChannel).build();
-    try {
-      assertThat(exporter.export(Collections.singletonList(generateFakeLog())).isSuccess())
-          .isFalse();
-    } finally {
-      exporter.shutdown();
-    }
-  }
-
-  @Test
-  void testExport_PermissionDenied() {
-    fakeCollector.setReturnedStatus(Status.PERMISSION_DENIED);
```

Comment:
The old tests had a bunch of status codes but I didn't copy them all since they didn't seem to test anything special